samples/corda/corda-simple-application/clients/src/main/kotlin/com/cordaSimpleApplication/client/UtilsManager.kt

```diff
@@ -0,0 +1,229 @@
+/*
+ * Copyright IBM Corp. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.cordaSimpleApplication.client
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.core.requireObject
+import com.github.ajalt.clikt.parameters.arguments.argument
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.flag
+import com.github.ajalt.clikt.parameters.options.default
+import java.io.File
+import java.lang.Exception
+import java.security.cert.X509Certificate
+import org.json.JSONObject
+import net.corda.core.messaging.CordaRPCOps
+import kotlinx.coroutines.runBlocking
+import kotlin.system.exitProcess
+import java.util.Base64
+import net.corda.core.identity.CordaX500Name
+import net.corda.core.identity.Party
+import com.weaver.corda.app.interop.flows.x509CertToPem
+
+class AssetUtilsCommand : CliktCommand(name = "util", help ="Manages asset utilities") {
```

Comment:
Can we remove `asset` from this name? these utility functions can be used anywhere not necessarily in assets.

Reply:
Done.